bot/util/utilities.py

```diff
@@ -154,24 +154,145 @@ async def cembed(self, ctx, channel: str, *, json_string: str):
         except discord.errors.HTTPException:
             await ctx.send("**__Error:__** Could not parse json. Make sure your last argument is valid JSON.")
         except discord.DiscordException:
-            await ctx.send(
-                "**__Error:__** Invalid embed. Make sure you have at least title or description set (also for each additional field). You can validate your json at https://leovoel.github.io/embed-visualizer/.")
+            await ctx.send("**__Error:__** Invalid embed. Make sure you have at least title or description set (also "
+                           "for each additional field). You can validate your json at "
+                           "https://leovoel.github.io/embed-visualizer/.")
         except ValueError as error:
             await ctx.send(error)
         except TypeError:
             await ctx.send("**__Error:__** Error creating embed. Please check your parameters.")
 
     @commands.command(name="echo")
-    async def echo(self, ctx, channel: typing.Optional[discord.TextChannel], *, text: str):
+    @commands.has_guild_permissions(administrator=True)
+    async def echo(self, ctx: commands.Context, channel: typing.Optional[discord.TextChannel], *, text: str):
         """Lets the bot post a simple message to the mentioned channel (or the current channel if none is mentioned).
 
         Args:
             ctx (discord.ext.commands.Context): The context from which this command is invoked.
             channel (Optional[str]): The channel where the message will be posted in.
-            text (str): The text to be echoed
+            text (str): The text to be echoed.
         """
         await (channel or ctx).send(text)
 
+    @commands.group(name="bot")
+    @commands.has_guild_permissions(administrator=True)
+    async def bot_cmd(self, ctx: commands.Context):
+        """Command handler for the `bot` command.
+
+        This is a command group regarding everything directly bot related. It provides a variety of subcommands for
+        special tasks like rebooting the bot or changing its Discord presence. For every single subcommand administrator
+        permissions are needed. If no subcommand has been provided, the corresponding help message will be posted
+        instead.
+
+        Args:
+            ctx (discord.ext.commands.Context): The context from which this command is invoked.
+        """
+        if ctx.invoked_subcommand is None:
+            await ctx.send_help(ctx.command)
+
+    @bot_cmd.group(name="presence")
+    async def change_discord_presence(self, ctx: commands.Context):
+        """Command handler for the `bot` subcommand `presence`.
+
+        This is a command group for changing the bots Discord presence. For every user-settable activity type there is
```

Comment:
Why do you describe the arguments of other commands here? They are described in the other commands' docstrings. You could just inform what other subcommands there are

Reply:
Good point. I will remove this.